resources/META-INF/plugin.xml

```diff
@@ -225,6 +225,7 @@
         <applicationService serviceImplementation="com.ansorgit.plugins.bash.editor.codecompletion.BashPathCompletionService" />
         <completion.contributor language="Bash" implementationClass="com.ansorgit.plugins.bash.editor.codecompletion.BashCompletionContributor"/>
 
+        <indexPatternBuilder implementation="com.ansorgit.plugins.bash.lang.psi.BashIndexPatternBuilder" />
```

Comment:
`dev-resources/META-INF/plugin.xml` is the file configured in the IDEA project. 
It's there because the test runner is not loading optional plugin dependencies (i.e. dependencies to the terminal plugin and IntelliLang).  At least that was the case when I made this change, things may have improved in the meantime.
That's why I'm keeping two files, one for development with hard dependencies and the one used for deployment with optional dependencies.

I agree that this could be more automated. Let me know if you want me to do that.

Reply:
OK, if it's easier to keep two files right now, let's use two, I don't mind.